eng/versioning/external_dependencies.txt

```diff
@@ -67,8 +67,8 @@ io.netty:netty-tcnative-boringssl-static;2.0.62.Final
 io.netty:netty-transport-native-epoll;4.1.100.Final
 io.netty:netty-transport-native-unix-common;4.1.100.Final
 io.netty:netty-transport-native-kqueue;4.1.100.Final
-io.projectreactor.netty:reactor-netty;1.0.38
-io.projectreactor.netty:reactor-netty-http;1.0.38
+io.projectreactor.netty:reactor-netty;1.0.39
+io.projectreactor.netty:reactor-netty-http;1.0.39
 io.projectreactor:reactor-core;3.4.33
```

Comment:
Upgrade this and Reactor Test to `3.4.34` to align with Reactor Netty dependencies.